app/templates/components/browse-list.html

```diff
@@ -5,7 +5,7 @@
         {% for item in items %}
           {% if item.title and item.link %}
             <li class="browse-list-item">
-              <a href="{{ item.link }}" class="govuk-link govuk-link--{% if item.destructive %}--destructive{% else %}--no-visited-state{% endif %}">{{ item.title }}</a>
+              <a href="{{ item.link }}" class="usa-link usa-link{% if item.destructive %}--destructive{% else %}--no-visited-state{% endif %}">{{ item.title }}</a>
```

Comment:
Just making sure I'm seeing this right:  it looks like there was an extraneous `--` here originally, which was likely incorrect, in addition to the class name changes?

Reply:
@ccostino correct - from what I can tell the "double" -- isn't necessary